target/linux/ipq40xx/image/generic.mk

```diff
@@ -759,6 +759,19 @@ define Device/qxwlan_e2600ac-c2
 endef
 TARGET_DEVICES += qxwlan_e2600ac-c2
 
+define Device/telco_x1-pro-lte
+	$(call Device/FitImage)
+	DEVICE_VENDOR := Telco
+	DEVICE_MODEL := X1 Pro
+	DEVICE_VARIANT := LTE
+	BOARD_NAME := x1-pro-lte
```

Comment:
I don't think BOARD_NAME is still required for anything.

Reply:
dropped